flake8_trio.py

```diff
@@ -65,6 +65,7 @@
         "Nurseries should generally be the inner-most context manager."
     ),
     "TRIO112": "Redundant nursery {}, consider replacing with directly awaiting the function call",
+    "TRIO113": "Dangerous `.start()`, process might not run before `__aenter__` exits. Consider replacing with `.start_soon()`.",

```

Comment:
This is reversed: we want to warn because `nursery.start_soon()` might not take effect during `__aenter__` and recommend `await nursery.start()`

Reply:
derp, brainfart. Fixed